style.css

```diff
@@ -1,62 +1,226 @@
-  /* position: fixed; */
+/* JS Navbar Scrolling CSS */
+.navbar.scrolled {
+	background: #a1cbdf;
+	transition: background 400ms;
 }
-
-.splash {
-  font-family: 'Noto Sans', sans-serif;
-  padding-top: 60px;
+#city {
+	background: url('building.png');
+	background-size: cover;
+	min-height: 100vh;
+	position: relative;
+}
+#BRAND {
+	font-size: 1.2rem;
+	padding-top: 0;
+	padding-bottom: 0;
+}
+#NL {
+	font-size: 1rem;
+	padding-top: 2px;
+	padding-bottom: 2px;
+}
+/* --------------------------------------- SECTION 1 --------------------------------------------------------------- */
+.nameDev {
+	margin-top: 0;
+	margin-bottom: 0;
+}
+.links {
+	color: #a0ffff;
+}
+.links:hover {
+	color: #a0ffff;
+}
+.socialList {
+	padding-top: 1rem;
+	text-align: center;
+	color: #a0ffff;
+	font-size: 1.5rem;
+}
+#github {
+	padding: 1rem;
+}
+#linkedin {
+	padding: 1rem;
+}
+#file {
+	padding: 1rem;
+}
+.toplinework {
+	/* this div holds all of the top line work*/
+	margin-bottom: 0px;
+	height: 50px;
+}
+.line {
+	border-top: 2px solid white;
+	margin-top: 5px;
+	margin-bottom: 5px;
+}
+#l1 {
+	width: 10%;
+}
+#l2 {
+	width: 25%;
+}
+#l3 {
+	width: 35%;
 }
 
-.picture {}
-
-#mePic {
-  margin-left: auto;
-  margin-right: auto;
-  max-width: 50%;
-  max-height: 50%;
-  /* max-width: 40%; */
-  background:#A0FFFF;
-  border-radius: 100%;
-  border-style: solid;
-  border-color: #A0FFFF;
-  border-width: 10px;
+#l4 {
+	width: 41%;
+}
+#l5 {
+	width: 44%;
+}
+#l6 {
+	width: 47%;
 }
 
+.thickLine {
+	margin-top: 5px;
+	margin-bottom: 0px;
+	border-top: 8px solid white;
+	width: 50%;
+	position: relative;
+}
 
-h1 {
-  font-weight: bolder;
-  color:#639E9E;
-  letter-spacing: 4.5px;
-  font-size: 50px;
-  padding-bottom: 9;
-  margin-bottom: 9;
-  text-emphasis-color: 50px;
-  
+.thickLine::before {
+	content: '';
+	display: block;
+	position: absolute;
+	height: calc(2.5rem + 1px);
+	top: calc(-2.9rem - 2px);
+	left: calc(50% - 1px);
+	width: 2px;
+	background: white;
 }
 
-h2 {
-  color:#FFFFA0;
-  font-size: 18px;
-  letter-spacing: 9px;
-  border: 5px solid #A0FFFF;
-  margin: 0;
-  padding: 0;
-  background-color: rgba(0, 0, 0, 0.2)
+.sH1 {
+	font-weight: bolder;
+	color: #a0ffff;
+	letter-spacing: 8px;
+	font-size: 3.5rem;
+	text-align: center;
+	margin-bottom: 0px;
+}
+.sH2 {
+	font-weight: bolder;
+	color: #cfffa0;
+	letter-spacing: 15px;
+	font-size: 2.5rem;
+	text-align: center;
+	margin-bottom: 0px;
 }
 
-.bio {
-  color: white;
-  font-size: 1em;
-  /* text-align: center; */
+.bottomlinework {
+	/* this div holds all of the bottom line work*/
+	transform: rotate(180deg);
+	margin-bottom: 0px;
+	height: 50px;
+}
+/* -------------------------------- SECTION 2 ----------------------------------------------------------- */
+#section2 {
+	background-color: #dae298;
+}
+/* ------------------------------------------------------------------------------------------------------ */
+/* iPhone MAX Media Query */
+@media screen and (max-width: 414px) {
```

Comment:
**Media Queries**
[Check Bootstrap for a guide](https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints)

The industry pattern is to write for *mobile first*, then add media queries using `min-width` as you grow.
Your CSS file (not that it doesn't work), is setting *upper* limits on content, whereas, the usual approach is to set a starting limit.

For example, you wouldn't need a `max-width: 320px`, you could instead have a style that starts at `min-width: 321px`. And another that starts at `max-width:415px`.

There can be lots of debate as to the "thresholds" of each break point, but if you stick with the bootstrap norms, especially since you're using the library, you should b good. They don't hit their first break point until `576px`

It's best not to be "device-specific" with breakpoints since not everyone has an iPhone Max.

Try to re-arrange your styles so that you don't need the `max-width: 320px` media query; your smallest screen styles should be the default. If your page reacts very differently from `320px` to `576px`, then you can add another as needed.

It's looking great!